### PR TITLE
feat: SWPG6SPEAK-254/255/256/261/263/264/265: Create message UI

### PR DESCRIPTION
**Pull request by sgatnar**
Created message ui

### CODE REVIEW

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 1]
```diff
@@ -1,4 +1,10 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:frontend/pages/base/messages/user.dart';
+import 'package:frontend/widgets/all_widgets.dart';
+import '../../../utils/speaq_styles.dart';
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:5]: Change to package import

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+
+//https://dev.to/iizmotabar/flutter-ui-series-whatsapp-clone-51e1
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:7]: Remove

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 3]
```diff
@@ -9,13 +15,178 @@ class MessagesPage extends StatefulWidget {
...
+            preferredSize: deviceSize,
+            centerTitle: true,
+            actionList: [generateSettingsIcon(context)],
+            leading: Builder(builder: (context) {
+              return IconButton(
+                  onPressed: () => Scaffold.of(context).openDrawer(),
+                  icon: CircleAvatar(
+                    radius: 20,
+                    backgroundImage: NetworkImage(profilePicture),
+                  ));
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:88]: Comma

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 4]
```diff
@@ -9,13 +15,178 @@ class MessagesPage extends StatefulWidget {
...
+                  ),
+                  title: Text(
+                    _foundUsersList[index].name,
+                  ),
+                  subtitle: Text(_foundUsersList[index].lastMessage),
+                  trailing: Text(_foundUsersList[index].time),
+                );
+              },
+            )),
+          ])),
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:110]: Commas

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 5]
```diff
@@ -9,13 +15,178 @@ class MessagesPage extends StatefulWidget {
...
+    return SizedBox(
+      height: deviceSize.height * 0.08,
+      width: deviceSize.width,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8, horizontal: 4),
+        child: TextField(
+          onChanged: (value) => filterSearchResults(value),
+          decoration: InputDecoration(
+            filled: true,
+            //contentPadding: EdgeInsets.symmetric(vertical: 0),
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:124]: Remove unnecessary comments

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 6]
```diff
@@ -9,13 +15,178 @@ class MessagesPage extends StatefulWidget {
...
+
+  filterSearchResults(String text) {
+    List<User> filterList = <User>[];
+    if (text != null) {
+      setState(() {
+        filterList.addAll(_allUserList
+            .where((user) => user.name.toString().contains(text))
+            .toList());
+
+        print(_allUserList
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:159]: Remove

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 7]
```diff
@@ -9,13 +15,178 @@ class MessagesPage extends StatefulWidget {
...
+
+
+    setState(() {
+      _foundedUsers = _userList
+          .where((user) => user.name.toLowerCase().contains(text))
+          .toList();
+    });
+    //print(_foundedUsers.toString());
+
+     */
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:192]: Remove unnecessary comments

File: frontend/lib/pages/base/messages/user.dart [hunk 8]
```diff
@@ -0,0 +1,10 @@
+class User{
+
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/user.dart:2]: Remove unnecessary line breaks

File: frontend/lib/pages/base/messages/user.dart [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+  final String profilePic;
+  final String name;
+  final String lastMessage;
+  final String time;
+
+  User(this. profilePic, this.name, this.lastMessage, this.time);
+
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/user.dart:9]: Remove unnecessary line breaks

File: frontend/lib/pages/settings/sub_settings/settings_account_page.dart [hunk 10]
```diff
@@ -1,10 +1,9 @@
 import 'package:flutter/material.dart';
-import 'package:frontend/widgets/speaq_appbar.dart';
-import 'package:frontend/widgets/speaq_settings_section.dart';
-
+import 'package:flutter_svg/svg.dart';
 import 'package:settings_ui/settings_ui.dart';
 
-import '../../../widgets/speaq_bottom_logo.dart';
+import '../../../widgets/speaq_appbar.dart';
+import '../../../widgets/speaq_settings_section.dart';
```
Comment (haveachin) [inline frontend/lib/pages/settings/sub_settings/settings_account_page.dart:6]: Change to package import

**Review by haveachin — DISMISSED** [review]

File: frontend/lib/pages/base/messages/messages_page.dart (hunk 3 above)
Comment (sgatnar) [reply frontend/lib/pages/base/messages/messages_page.dart:88]: done

File: frontend/lib/pages/base/messages/messages_page.dart (hunk 4 above)
Comment (sgatnar) [reply frontend/lib/pages/base/messages/messages_page.dart:110]: done

File: frontend/lib/pages/base/messages/messages_page.dart (hunk 5 above)
Comment (sgatnar) [reply frontend/lib/pages/base/messages/messages_page.dart:124]: done

File: frontend/lib/pages/base/messages/messages_page.dart (hunk 6 above)
Comment (sgatnar) [reply frontend/lib/pages/base/messages/messages_page.dart:159]: done

File: frontend/lib/pages/base/messages/messages_page.dart [hunk 11]
```diff
@@ -9,13 +13,158 @@ class MessagesPage extends StatefulWidget {
...
+  filterSearchResults(String text) {
+    List<User> filterList = <User>[];
+    if (text != null) {
+      setState(
+        () {
+          filterList.addAll(_allUserList
+              .where((user) => user.name.toString().contains(text))
+              .toList());
+          _foundUsersList = filterList;
+        },
```
Comment (haveachin) [inline frontend/lib/pages/base/messages/messages_page.dart:164]: Remove that comma

**Review by haveachin — CHANGES_REQUESTED** [review]

**Review by haveachin — APPROVED** [review]

**Pull request merged**